elasticsearch/elasticsearch.d.ts

```diff
@@ -102,6 +104,21 @@ declare module Elasticsearch {
         index?: string;
     }
 
+    export interface ClientCreateParams extends GenericParams {
```

Comment:
rename interface to CreateDocumentParams. Its more similar to the other document operation input parameters.